README.md

```diff
@@ -1,2 +1,39 @@
-# perl-Test-Pod-CoverageChange
-Wraps Test::Pod::Coverage to support undocumented subs and statistics on changed coverage
+# NAME
+
+Wraps Test::Pod::Coverage and Pod::Checker modules to support undocumented subs and statistics on changed coverage.
```

Comment:
The NAME section needs to have the module name - and then a quick description if needed. See e.g. `man DBI`